pyanaconda/kickstart.py

```diff
@@ -80,7 +80,7 @@
                                   SELINUX_DISABLED, SELINUX_ENFORCING, SELINUX_PERMISSIVE, \
                                   SNAPSHOT_WHEN_POST_INSTALL, SNAPSHOT_WHEN_PRE_INSTALL
 from pykickstart.base import BaseHandler, KickstartCommand
-from pykickstart.errors import formatErrorMsg, KickstartError, KickstartValueError
+from pykickstart.errors import formatErrorMsg, KickstartError, KickstartValueError, KickstartParseError
```

Comment:
You should use `KickstartValueError` for rhel7-branch as all the other commands do.

Reply:
Any real reason for this? I mean, it makes no difference and only makes porting/cherry-picking things harder.

Reply:
The main reason here is the consistency. Yeah I know the difference on rhel and master is a pain but I don't want to introduce inconsistency because of harder cherry-picking.  

On the `master` it is because deprecation of `KickstartValueError` in Pykickstart and RHEL used this exception before the deprecation.

But I'm open for discussion about changing all of those exception to `KickstartParseError` to make it consistent between rhel7 and master branches. What do you think @poncovka , @rvykydal , @M4rtinK ?

Reply:
I agree with @jkonecny12 . `KickstartParseError` doesn't make sense here in a context of RHEL 7.
```

class KickstartParseError(KickstartError):
    """An exception class for errors when processing the input file, such as
       unknown options, commands, or sections.
    """

class KickstartValueError(KickstartError):
    """An exception class for errors when processing arguments to commands,
       such as too many arguments, too few arguments, or missing required
       arguments.
    """
```

Reply:
For this PR I agree with Jirka and I'd just use KickstartValueError in rhel7-branch.
I wouldn't do the master/rhel7-branch unification in scope of this patch, but if Vratislav prefers to do that, I won't NACK it.

Reply:
So please @vpodzime change this to `KickstartValueError` and you have my ACK.